Popular Movies/Popular Movies/Controllers/MoviesCollectionViewController.h

```diff
@@ -0,0 +1,12 @@
+//
+//  MoviesCollectionViewController.h
+//  Popular Movies
+//
+//  Created by Test on 14/10/2017.
+//  Copyright © 2017 Test. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
```

Comment:
**Suggestion:** If you inherit **UICollectionViewController** class it is not needed to implement **UICollectionViewDelegate** and **UICollectionViewDataSource** protocols. This is needed when you inherit **UIViewController** class.